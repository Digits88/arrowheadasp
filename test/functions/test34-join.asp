<!-- Test Join function.
-->

Dim A1, A2, A3
	<% Dim A1, A2, A3 %><br>

A1 = Array(1, 2, 3)
	<% A1 = Array(1, 2, 3) %><br>

A2 = Array("Hello", "There")
	<% A2 = Array("Hello", "There") %><br>

A3 = Array()
	<% A3 = Array() %><br>

Join(A1) = <%= Join(A1) %><br>
Join(A2) = <%= Join(A2) %><br>
Join(A3) = <%= Join(A3) %><br>
Join(A1, ",") = <%= Join(A1, ",") %><br>
Join(A2, ",") = <%= Join(A2, ",") %><br>
Join(A3, ",") = <%= Join(A3, ",") %><br>
Join(A1, "cha") = <%= Join(A1, "cha") %><br>
Join(A2, "cha") = <%= Join(A2, "cha") %><br>
Join(A3, "cha") = <%= Join(A3, "cha") %><br>
