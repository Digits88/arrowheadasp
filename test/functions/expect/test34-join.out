<!-- Test Join function.
-->

Dim A1, A2, A3
	<br>

A1 = Array(1, 2, 3)
	<br>

A2 = Array("Hello", "There")
	<br>

A3 = Array()
	<br>

Join(A1) = 1 2 3<br>
Join(A2) = Hello There<br>
Join(A3) = <br>
Join(A1, ",") = 1,2,3<br>
Join(A2, ",") = Hello,There<br>
Join(A3, ",") = <br>
Join(A1, "cha") = 1cha2cha3<br>
Join(A2, "cha") = HellochaThere<br>
Join(A3, "cha") = <br>
